.github/workflows/nightly_docs.yml

```diff
@@ -17,6 +17,8 @@ jobs:
         run: rustup update
       - name: Rust Cache
         uses: Swatinem/rust-cache@v2.2.1
+      - name: Pin dependencies for MSRV
```

Comment:
This shouldn't be needed. I was able to build the locally docs (`cargo doc --no-deps`) with `nightly-2022-12-14` without any errors.

Reply:
Publish Nightly Docs / Build docs will unfortunately fail without this change.

> error: package `home v0.5.9` cannot be built because it requires rustc 1.70.0 or newer, while the currently active rustc version is 1.68.0-nightly
Either upgrade to rustc 1.70.0 or newer, or use
cargo update -p home@0.5.9 --precise ver
where `ver` is the latest version of `home` supporting rustc 1.68.0-nightly
Error: Process completed with exit code 101.

Reply:
That's strange, but anyway not a big deal, more important to get the rest of this PR in. Thanks for checking.